lib/mutant/mutator/node/block.rb

```diff
@@ -36,7 +36,32 @@ def mutate_body
           return unless body
           emit(body)
           emit_body_mutations
+
+          mutate_body_receiver
+        end
+
+        # Mutate method send in body scope of `send`
+        #
+        # @return [undefined]
+        #
+        # @api private
+        def mutate_body_receiver
+          return unless n_send?(body)
+
+          body_meta = AST::Meta::Send.new(body)
```

Comment:
OT: I wanted to automate the process of "getting the meta of a given AST node" in a way we:

* Do not duplicate all the `children` DSL calls per node (right now we do in unparser, mutant, mutan-ast-meta)
* Ideally only create ONE meta per node

This is an ongoing refactoring idea, for now lets backfill meta "instances" till I've time to PR it.